pkg/print/v1pb.go

```diff
@@ -0,0 +1,50 @@
+package print
+
+import (
+	"fmt"
+	"io"
+	"strings"
+	"time"
+
+	"github.com/go-phorce/trusty/api/v1/trustypb"
+	"github.com/olekukonko/tablewriter"
+)
+
+// ServerVersion prints ServerVersion
+func ServerVersion(w io.Writer, r *trustypb.ServerVersion) {
+	fmt.Fprintf(w, "%s (%s)\n", r.Build, r.Runtime)
+}
+
+// ServerStatusResponse prints trustypb.ServerStatusResponse
+func ServerStatusResponse(w io.Writer, r *trustypb.ServerStatusResponse) {
+
+	table := tablewriter.NewWriter(w)
+	table.SetBorder(false)
+	table.SetAlignment(tablewriter.ALIGN_LEFT)
+	table.Append([]string{"Name", r.Status.Name})
+	table.Append([]string{"Node", r.Status.Nodename})
+	table.Append([]string{"Host", r.Status.Hostname})
+	table.Append([]string{"Listen URLs", strings.Join(r.Status.ListenURLs, ",")})
+	table.Append([]string{"Version", r.Version.Build})
+	table.Append([]string{"Runtime", r.Version.Runtime})
+
+	startedAt := time.Unix(r.Status.StartedAt, 0)
+	uptime := time.Now().Sub(startedAt) / time.Second * time.Second
```

Comment:
@dissoupov could you remind why here we divide by time.Second and then multiply by the same time.Second?

Reply:
to round up `Uptime`  to the sec. So output will be:
`1h`, or `1h37m` or `1h37m15s` or `15s`
and ms will be dropped